gst-libs/mfx/gstmfxallocator_d3d11.c

```diff
@@ -49,7 +49,7 @@ gst_mfx_task_frame_alloc(mfxHDL pthis, mfxFrameAllocRequest * request,
   if (task->task_type & (GST_MFX_TASK_VPP_OUT | GST_MFX_TASK_DECODER))
   request->Type |= 0x1000;*/
 
-  if (priv->saved_responses && priv->task_type & GST_MFX_TASK_DECODER) {
+  if (priv->saved_responses && (priv->task_type & GST_MFX_TASK_DECODER) == GST_MFX_TASK_DECODER) {
```

Comment:
Forgot to mention, videos whose dimensions are not aligned to 16 cannot render in video memory (1080p)!